rsts/getting_started.sh.rst

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+:'
+
+.. _gettingstarted:
+
+Getting started
+---------------
+
+.. rubric:: Estimated time to complete: 3 minutes.
+
+
+Prerequisites
+***************
+
+Make sure you have `docker installed <https://docs.docker.com/get-docker/>`__ and `git <https://git-scm.com/>`__ installed, then install flytekit:
+
+Steps
+*****
+
+'
+
+# #. First install the python Flytekit SDK (maybe in a virtual environment) and clone the `flytekit-python-template <https://github.com/flyteorg/flytekit-python-template>`_ repo::
+
+  pip install flytekit
+  git clone git@github.com:flyteorg/flytekit-python-template.git myflyteapp
```

Comment:
may be use 
```
https://github.com/flyteorg/flytekit-python-template.git
```
which doesn't require to setup keys

Reply:
that is good idea